src/Resolver.js

```diff
@@ -185,4 +185,8 @@ export default class Resolver {
       return React.renderToStaticMarkup(context);
     });
   }
+
+  static decorate(props = {}) {
```

Comment:
ah yeah. I see how @ericclemmons’ doing it.
https://github.com/ericclemmons/react-resolver/blob/master/src/Resolver.js#L147